crates/contracts/build.rs

```diff
@@ -298,6 +298,9 @@ fn main() {
             .add_network_str("4", "0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D")
             .add_network_str("5", "0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D")
     });
+    generate_contract_with_config("UniswapV3SwapRouter", |builder| {
+        builder.add_network_str("1", "0xE592427A0AEce92De3Edee1F18E0157C05861564")
```

Comment:
Can we also support Goerli (UniV3 is deployed there AFAIK).

Reply:
Thanks. Added.